src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -299,13 +323,34 @@ export default class OrganizationDiscover extends React.Component {
               />
             )}
             {!data && <Intro updateQuery={this.updateFields} />}
-          </Box>
-        </Flex>
+            <EarlyAdopterMessage />
+          </Flex>
+        </DiscoverBody>
       </Discover>
     );
   }
 }
 
+const DiscoverHeading = styled.h2`
+  font-size: 18px;
+  font-weight: normal;
+  color: ${p => p.theme.gray4};
+  margin: 0;
+`;
+
+const SidebarHeader = styled(Flex)`
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+`;
```

Comment:
Should these be styled like the sidebar on Issues?

<img width="329" alt="screen shot 2018-09-04 at 10 39 11 am" src="https://user-images.githubusercontent.com/24086/45038236-5b463480-b050-11e8-9596-ca7e1f5b5ada.png">

Reply:
sorry for the delay! since this is a new and different app we have separate styling specs for this.